openadapt/app/dashboard/components/ActionEvent/ActionEvent.tsx

```diff
@@ -0,0 +1,151 @@
+'use client';
+
+import { timeStampToDateString } from '@/app/utils';
+import { ActionEvent as ActionEventType } from '@/types/action-event'
+import { Accordion, Box, Grid, Table } from '@mantine/core'
+import Image from 'next/image';
+
+type Props = {
+    event: ActionEventType,
+    values: string[],
+    level: number,
+}
+
+export const ActionEvent = ({
+    event,
+    values,
+    level,
```

Comment:
It would be useful to have some sort of tree view here, since it's not immediately clear what is the hierarchy. Compare e.g.:
<img width="273" alt="image" src="https://github.com/OpenAdaptAI/OpenAdapt/assets/774615/23d75a6a-1e88-435d-ac04-2e8ca61acbd1">


Perhaps just increasing the left margin and adding a vertical line would be sufficient, but ideally we would use some tree component, e.g. https://github.com/react-component/tree